python/foglamp/services/core/api/configuration.py

```diff
@@ -505,7 +505,7 @@
 
     cf_mgr = ConfigurationManager(connect.get_storage_async())
     try:
-        await cf_mgr.delete_parent_category(category_name)
+        await cf_mgr.delete_recursively_parent_category(category_name)
```

Comment:
I think its main use case is for delete service. We need to call this in api/service.py>delete_service and remove redundant calls from there.